stable/redis/values.yaml

```diff
@@ -0,0 +1,33 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis:3.2.4-r0
```

Comment:
Good question, although it's not being used here, the only additional feature is that the Bitnami Redis image supports a replication mode. Also the [docs](https://github.com/bitnami/bitnami-docker-redis) are more thorough than the official Redis image.

More importantly though, as we maintain both the Chart and the image, we're focused on creating the best experience for this Chart.